### DISCUSSION TITLE
Testnet

### DISCUSSION THREAD

**Discussion by Muzair217** [blockscout/blockscout#9913] [Show and tell]
Hi i successfully kyc but i not receive testnet 10 usdt why ???

**Comment by Muzair217** [comment]
![Uploading Screenshot_2024-04-17-17-11-53-95_5829d8aa5c1dd419e78b999d5ba09c49.jpg…]()

**Reply by Thanawat2522** [reply]
I transferred coins 36 hours ago, 10 usdt, but why haven't the coins arrived in my wallet?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
